packages/neon_framework/pubspec.yaml

```diff
@@ -13,7 +13,6 @@ dependencies:
   cookie_jar: ^4.0.0
   crypto: ^3.0.0
   crypton: ^2.0.0
-  cupertino_icons: ^1.0.0
```

Comment:
Meh I don't like that it is used so indirectly. I wonder how well the icon tree shaker handles this. Nothing we can do though.

Reply:
DO you mean on non iOS platforms?
There the icons aren't even included because of licensing issues.

The apple font and accompanied icons are only allowed to be used on iOS (and probably also macOS?). Therefore setting the platform in the theme to `iOS` the icon isn't even visible (iirc we already talked about this when I landed the changes).

Don't we love apple :pray:

Reply:
So they have a platform specific license?? Jesus that is stupid...